### PR TITLE
Docs/refactors variables topics

### PR DESCRIPTION
**Pull request by chri2547**
This PR refactors add variables topics.

### CODE REVIEW

File: docs/sources/variables/add-template-variables/index.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+1. Enter a **Name** for the variable.
+1. In the **Type** list, select **Query**.
+1. (Optional) In **Label**, enter the display name of the variable dropdown.
+
+   If you don't enter a display name, then the dropdown label is the variable name.
+
+1. Choose a **Hide** option:
+   - **No selection (blank):** The variable dropdown displays the variable **Name** or **Label** value. This is the default.
+   - **Label:** The variable dropdown only displays the selected variable value and a down arrow.
+   - **Variable:** No variable dropdown is displayed on the dashboard.
```
Comment (torkelo) [inline docs/sources/variables/add-template-variables/index.md:81]: These general options looks duplicated a bunch in this file. For this first section it also looks a bit strange interms of spacing (lots of empty space between bullets) 

Screenshot from 2022-08-31 15-57-32

File: docs/sources/variables/add-template-variables/index.md [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+
+### Enter Query Options
```
Comment (torkelo) [inline docs/sources/variables/add-template-variables/index.md:83]:
```suggestion
### Enter query options
```

Sentance case for the win :)

Comment (chri2547) [reply docs/sources/variables/add-template-variables/index.md:83]: Good catch, we'll make you an Editor after all! :)

File: docs/sources/variables/add-template-variables/index.md (hunk 1 above)
Comment (chri2547) [reply docs/sources/variables/add-template-variables/index.md]: stepped out the general options as its own section and then refer back to it in step 1 of every task. Was able to remove the duplicate content.

**Review by torkelo — APPROVED** [review]
Screenshot from 2022-09-09 08-37-47

The main topic name still feels too wordy (at least in the menu). In the UI we now only refer these as "Variables" (Used to be called Template variables, but simplified it a bit). I think it's ok for the docs to call them by their full name in the main section but does not need to repeat the full title everywhere. 

Some options (For the menu)
* Manage variables  
* Define variables 
* Add and edit variables

I am also missing section and link from the index page so it becomes easy find the main content page from the index page.
